src/ivoc/ivoc.cpp

```diff
@@ -373,12 +373,9 @@ int Oc::run(const char* buf, bool show_err_mes) {
         err = hoc_oc(buf);
     } catch (std::exception const& e) {
         if (show_err_mes) {
-            Fprintf(stderr, "Oc::run: caught exception");
             std::string_view what{e.what()};
-            if (!what.empty()) {
-                Fprintf(stderr, fmt::format(": {}", what).c_str());
-            }
-            Fprintf(stderr, "\n");
+            std::string message = what.empty() ? "" : fmt::format(": {}", what);
```

Comment:
This looks risky, the two branches have different types: `char[]` and `std::string`. IIRC this can cause problems.

Reply:
"In C++14, C++17 or C++20, you can place an s after the quotes, and it will create a std::string instead of a const char* string."
```suggestion
            std::string message = what.empty() ? ""s : fmt::format(": {}", what);
```

Reply:
Done, thanks